eclass/bitcoincore.eclass

```diff
@@ -340,6 +340,7 @@ bitcoincore_prepare() {
 }
 
 bitcoincore_autoreconf() {
+	[ "${EAPI}" != 5 ] && eapply_user
```

Comment:
EAPIs are not necessarily numbers, and so matching on `>` is not guaranteed to give sane results.